test/DynamoCoreTests/Performance/PerformanceTests.cs

```diff
@@ -134,5 +224,107 @@ private void CheckForDummyNodes(WorkspaceModel ws)
                 Assert.Inconclusive("The Workspace contains dummy nodes for: " + string.Join(",", dummyNodes.Select(n => n.Name).ToArray()));
             }
         }
+
+        private void LotsOfColoredStuffSimplified()
+        {
+            var seeds = Enumerable.Range(0, 26).Select(x => x * 2).ToList();
+
+            var points = seeds.Select(x => seeds.Select(y => seeds.Select(z => Point.ByCoordinates(x, y, z)).ToList()).ToList()).ToList();
```

Comment:
hmm, do you think all this linq is similar in performance to a bunch of for loops?

Reply:
I have assumed that it is similar but I have no proof. I found it easier to use and reason about. That being said, it should not be that much more work to turn this into a bunch of for loops now when I have locked down the results.